### ISSUE TITLE
Provide example on how to run as CronJob

### ISSUE THREAD

**Issue by hjacobs** [hjacobs/kube-janitor#1]
Running `kube-janitor` as a CronJob is probably better (does not need to run very frequently).

**Comment by SongGithub** [comment]
yep, it can be simply done by create a cronjob with cmd `kube-janitor ... --once`
